src/controllers/__tests__/profileController.test.ts

```diff
@@ -20,7 +19,7 @@ const aFiscalNumber = "GRBGPP87L04L741X" as FiscalCode;
 const anEmailAddress = "garibaldi@example.com" as EmailAddress;
 const anIsInboxEnabled = true as IsInboxEnabled;
 const anIsWebookEnabled = true as IsWebhookEnabled;
-const aPreferredLanguage = "it_IT" as PreferredLanguage;
+const aPreferredLanguages: ReadonlyArray<any> = ["it_IT"];
```

Comment:
you should use the [PreferredLanguages](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/api/definitions.yaml#L303) type here